config/application.php

```diff
@@ -77,6 +77,19 @@ function env( string $key ) : string {
 define( 'LOGGED_IN_SALT', env( 'LOGGED_IN_SALT' ) );
 define( 'NONCE_SALT', env( 'NONCE_SALT' ) );
 
+/**
+ * Multisite setup
+ */
+if ( env( 'IS_MULTISITE' ) ) {
+	define( 'WP_ALLOW_MULTISITE', true );
+	define( 'MULTISITE', true );
+	define( 'SUBDOMAIN_INSTALL', false );
```

Comment:
Should we add an variable for subdomain with default to false?

Reply:
Yes, good suggestion. Fixed in next commit